.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
We typically trigger on pull requests and push to master

Reply:
fixed.